### ISSUE TITLE
s3_wrapper only handles hdf4, should generalize.

### ISSUE THREAD

**Issue by michaelleerilee** [SpatioTemporal/STAREPandas#97]
The s3_wrapper gives convenient s3 access via s3fs, but it is currently hardwired for hdf4 reads. It should be generalized to handle other types, if it is to be kept.

**Comment by NiklasPhabian** [comment]
yes; this should not be too hard. Will probably push this off until we have a usecase or a particular demand.
